CHANGES.rst

```diff
@@ -6,9 +6,8 @@ Internals
 
 * now `Expression.is_numeric()` accepts an `Evaluation` object as a parameter,
```

Comment:
If the only think needed are the definitions, then why narrow the parameter to just the definition? 

@mmatera Your thoughts?

Reply:
@rocky, I just leave this in case we want to delegate something to the evaluation object, but maybe it would be clearer if the parameter is the `Definitions` object.